### ISSUE TITLE
Electron Win32 on 32bit architecture not working

### ISSUE THREAD

**Issue by arifbutt1964** [Urucas/electron-packager-interactive#16]
Hi,
I have downloaded the Electron windows version using the  electron package interactive, the epi created 2 folders: 
"electron_tutorial_one-win32-ia32"
"electron_tutorial_one-win32-x64"

The x64 is working OK on windows 64 bit machines but the ia32 is not working on 32 bit machines, giving an error that it is not compatible, see the attached file error description.

![error comdeck](https://cloud.githubusercontent.com/assets/20858615/17435839/ca9ae5c2-5b2c-11e6-8e2f-505962c220f7.jpg)

Please assist.

Regards
Arif

**Comment by arifbutt1964** [comment]
Please see the "electron_tutorial_one-win32-ia32" error in the attached file:
![xms](https://cloud.githubusercontent.com/assets/20858615/17436105/85c28e76-5b2e-11e6-8b6c-2c97720b4b44.jpg)

Thanks
Arif

**Comment by vrunoa** [comment]
hey @arifbutt1964 please verify on the `epi` output if it is correctly selecting *win 32*. Also ask/verify `electron-packager` is giving support to win32 while i'll try to investigate a bit more

**Comment by arifbutt1964** [comment]
Hi,
Yes the epi is selecting the correct output and making the 2 prebuilt 
"electron_tutorial_one-win32-ia32"
"electron_tutorial_one-win32-x64", 

I thought this page belongs to epi, however kindly let me know the electron-packager support contact so I can post them the same...

Thanks
Regards
Arif

**Comment by vrunoa** [comment]
Yeah @arifbutt1964 this page belongs to `epi` but i'm feelling this might be an error on `electron-packager`, since `epi` it's just a wrapper of `electron-packager` [https://github.com/electron-userland/electron-packager](https://github.com/electron-userland/electron-packager)
